### DISCUSSION TITLE
How does one escape the restore part of the container?

### DISCUSSION THREAD

**Discussion by devanteweary** [JonathanTreffler/backblaze-personal-wine-container#89] [Q&A]
I can restore files within the container by clicking Restore Options > Restore App.
However, once that loads, I have no idea how to get back to the main Backblaze window.

How to I exit the Restore part or switch windows?

Thank you!

(using Unraid).

**Comment by mason1171** [comment]
Alt-F4 works for me in my vnc viewer
